src/store/actions.ts

```diff
@@ -157,3 +158,33 @@ export const initWallet = () => {
         }
     };
 };
+
+export const cancelOrder = (order: SignedOrder) => {
+    return async (dispatch: any, getState: any) => {
+        const state = getState();
+        const ethAccount = getEthAccount(state);
+
+        const web3Wrapper = await getWeb3WrapperOrThrow();
+        const networkId = await web3Wrapper.getNetworkIdAsync();
+        const contractWrappers = new ContractWrappers(web3Wrapper.getProvider(), { networkId });
+        const tx = await contractWrappers.exchange.cancelOrderAsync(order, {
+            gasLimit: 1000000,
+        });
+        await web3Wrapper.awaitTransactionSuccessAsync(tx);
+
+        const relayer = getRelayer();
```

Comment:
Maybe we can move the "load orders" logic to another action, and dispatch it both from here and from the init action?

Reply:
Done with this little refactor.